PYME/Acquire/Hardware/AndorIXon/AndorControlFrame.py

```diff
@@ -73,15 +73,15 @@ def _createCollapsingPane(self):
         self.rbSingleShot.SetValue(False)
         #self.rbSingleShot.SetToolTipString('Allows multiple channels with different integration times and good shutter synchronisation')
         self.rbSingleShot.Bind(wx.EVT_RADIOBUTTON,self.OnRbSingleShotRadiobutton)
-        sbAqMode.Add(self.rbSingleShot, 1, wx.EXPAND|wx.ALIGN_CENTER_VERTICAL, 0)
```

Comment:
I should have mentioned the tricky bit on the initial issue - it's only an error if being added to a `BoxSizer` (or `StaticBoxSizer` that was created as `wx.BoxSizer(wx.VERTICAL)`. As `sbAqMode` is a horizontal sizer, the `wx.ALIGN_CENTER_VERTICAL` flag is OK (and highly desirable - the GUI looks very unpolished if it's absent).

Likewise the `ALIGN_CENTER_HORIZONTAL` flag on a vertical sizer. Most of the usages of these flags are not an error, it's just the few where we've either changed from a horizontal sizer to a vertical (or vice-versa), or have copy-pasted sizer additions without thinking about what type of sizer we are adding to.

Reply:
Oops, posted this without reading the `wx.EXPAND` bit. Need to think a bit more about it.